### ISSUE TITLE
v0.20.4 binary does not accept new flag --discovering-mode

### ISSUE THREAD

**Issue by caiorcferreira** [percona/mongodb_exporter#267]
When running the binary with the latest release v0.20.4 it fails when providing the `--discovering-mode` with the following message:

```
Usage: mongodb_exporter

MongoDB Prometheus exporter

mongodb_exporter: error: unknown flag --discovering-mode
Flags:
  -h, --help                      Show context-sensitive help.
      --mongodb.collstats-colls=db1.col1,db2.col2
                                  List of comma separared databases.collections
                                  to get $collStats
      --mongodb.indexstats-colls=db1.col1,db2.col2
                                  List of comma separared databases.collections
                                  to get $indexStats
      --mongodb.uri=mongodb://user:pass@127.0.0.1:27017/admin?ssl=true
                                  MongoDB connection URI ($MONGODB_URI)
      --mongodb.global-conn-pool
                                  Use global connection pool instead of creating
                                  new pool for each http request.
      --mongodb.direct-connect    Whether or not a direct connect should be
                                  made. Direct connections are not valid if
                                  multiple hosts are specified or an SRV URI is
                                  used.
      --web.listen-address=":9216"
                                  Address to listen on for web interface and
                                  telemetry
      --web.telemetry-path="/metrics"
                                  Metrics expose path
      --log.level="error"         Only log messages with the given severuty or
                                  above. Valid levels: [debug, info, warn,
                                  error, fatal]
      --disable.diagnosticdata    Disable collecting metrics from
                                  getDiagnosticData
      --disable.replicasetstatus
                                  Disable collecting metrics from
                                  replSetGetStatus
      --compatible-mode           Enable old mongodb-exporter compatible metrics
      --version                   Show version and exit
```

Seems like the CLI don't even know the flag, although the code for it is on the main branch and the feature is mentioned on the [release notes](https://github.com/percona/mongodb_exporter/releases/tag/v0.20.4)

**Comment by caiorcferreira** [comment]
Any estimate for when this new release can be launched?

**Comment by percona-csalguero** [comment]
I am working in a release. Cannot tell you the release date yet.
I'll keep you informed

**Comment by percona-csalguero** [comment]
Hi. 
Version [0.20.5](https://github.com/percona/mongodb_exporter/releases/tag/v0.20.5) has been released and it is going to be part of PMM 2.18 in around 2 weeks.
You can also talk and ask questions in PMM's discord [here](https://discord.gg/hQBrHgGX)

Regards
